### ISSUE TITLE
Kneeboard doesn't show elevation for strike targets

### ISSUE THREAD

**Issue by FR0STKRIEGER** [dcs-liberation/dcs_liberation#1445]
I'm not sure if this is supposed to be in feature requests, but:

In strike-packages, the kneeboards show the coordinates for the strike targets. This is very useful for setting up JDAM's, SLAM's, JSOW's etc., but the page doesn't show the elevation of the targets, so you still need to find this in ME or F10 map.

The coordinates added a lot of immersion, especially in VR, since you can punch in the coords during cold start and stay immersed, in stead of a) looking through F10 or b) lifting your VR headset to look through your hand written notes. 

TL;DR: Having the elevation (in feet) would allow us to stay immersed and away from ME/F10 while doing strike missions.

Thanks for all the work you do, guys!

**Comment by DanAlbert** [comment]
If you can tell us where to get the elevation data we'd love to do this, but this isn't possible with any freely available data afaik.

Closing as infeasible, but if someone can point us at _how_ to do this, it'll be reopened.
